### PR TITLE
GlobalMarineNet.XWireless version 3.85

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107503)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/g/GlobalMarineNet/XWireless/3.85/GlobalMarineNet.XWireless.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: GlobalMarineNet.XWireless
+PackageVersion: 3.85
+Installers:
+- InstallerUrl: http://downloads.globalmarinenet.com/downloads/xwireless3.85-2.01.exe
+  Architecture: x86
+  InstallerType: exe
```
Comment (stephengillie) [inline manifests/g/GlobalMarineNet/XWireless/3.85/GlobalMarineNet.XWireless.installer.yaml:9]:
```suggestion
  InstallerType: portable
```

**Comment by stephengillie** [comment]
I mistook the installer for a portable application. This seems like a custom installer that's launched by a Nullsoft installer - using the `/S` switch caused the outer Nullsoft installer to not show, but the custom installer still appeared. All other switches seem to launch the installer as normal.

File: manifests/g/GlobalMarineNet/XWireless/3.85/GlobalMarineNet.XWireless.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: GlobalMarineNet.XWireless
+PackageVersion: 3.85
+Installers:
+- InstallerUrl: http://downloads.globalmarinenet.com/downloads/xwireless3.85-2.01.exe
+  Architecture: x86
+  InstallerType: portable
```
Comment (stephengillie) [inline manifests/g/GlobalMarineNet/XWireless/3.85/GlobalMarineNet.XWireless.installer.yaml:9]:
```suggestion
  InstallerType: nullsoft
```

**Comment by upintheairsheep** [comment]
This package installs a portable package, and opens it. It’s an installer for GMN products, an App Store basically

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm stuck here:
image

First installer installs with `/S` but it also contains another installer which launches as soon as the the first installer has finished which doesn't seem to have any silent switches.

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by ItzLevvie** [comment]
[Policy] Interactive-Only-Installer

**Pull request closed without merging**